WooCommerce/Classes/ViewModels/Order Details/OrderDetailsResultsControllers.swift

```diff
@@ -181,11 +174,14 @@ final class OrderDetailsResultsControllers {
 
     func update(order: Order) {
         self.order = order
-        // Product variation results controller depends on order items to load variations,
+        // Product and variation results controller depends on order items to load variations,
         // so we need to recreate it whenever receiving an updated order.
         self.productVariationResultsController = getProductVariationResultsController()
+        self.productResultsController = createProductResultsController()
+        self.productResultsController = createProductResultsController()
```

Comment:
nit: any reasons having two of the same lines?
```suggestion
        self.productResultsController = createProductResultsController()
```

Reply:
Thank you, this was added by mistake.